lib/dynflow/persistence_adapters/abstract.rb

```diff
@@ -46,6 +46,10 @@ def find_execution_plan_counts(options = {})
         filter(:execution_plan, options[:filters]).count
       end
 
+      def find_execution_plan_counts_after(options = {}, timestamp)
+        filter(:execution_plan, options[:filters]).filter(::Sequel.lit('ended_at >= ?', timestamp)).count
```

Comment:
The abstract adapter shouldn't rely on any implementation-specific details (ie. `Sequel.lit`). If that can't be done, it is better to just raise `NotImplementedError`